jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenter.java

```diff
@@ -130,6 +134,23 @@ protected DataSetReadyCallback getDataSetReadyCallback(final Integer startRange,
                 .withEmptyResultsCallback(() -> setEmptyResults());
     }
 
+    private Set<Long> findDuplicateId(LinkedList<TaskSummary> taskSummaries) {
```

Comment:
These two methods are not needed if you use a Set rather than a list
In order a Set implementation to work (the one I recommend is LinkedHashSet) you need to modify class TaskSummary and add equals and hashcode methods implementation based on the id member.

Reply:
I think we keep using  list is better. 
1 The code will compile to js code, so list or set will same.
2 if  we change to Set, we need change many related codes[1][2].
3 The feature does  is just not deduplication,[3], if taskList has duplicate taskId, we also need to check the slastatus, if it's PENDING, the code will remove,and keep another status. 
So, I think Set might not better way.


[1] https://github.com/kiegroup/jbpm-wb/blob/61e267bc82fc31ef03d01fb8806b4522583e232c/jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenter.java#L192
[2] https://github.com/kiegroup/jbpm-wb/blob/3ece99ba83ee6be48c44a6c567da54aa8f27dba6/jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListView.java#L94
[3] https://github.com/kiegroup/jbpm-wb/pull/1582/files#diff-9ce368cd261c74e47c039a079e4039d57cff8386f1e7ac419d0e3f78ad3436f7R144